support/support.go

```diff
@@ -520,3 +520,62 @@ func sanitizeEnv(kvs map[string]string) {
 		}
 	}
 }
+
+func Summarize(b *Bundle) (msgs []string) {
+	if b == nil {
+		return []string{}
+	}
+	if b.Network.Netcheck == nil {
+		msgs = append(msgs, "Netcheck missing from bundle!")
+	} else {
+		if b.Network.Netcheck.Error != nil {
+			msgs = append(msgs, "Client netcheck: "+*b.Network.Netcheck.Error)
+		}
+		for _, warn := range b.Network.Netcheck.Warnings {
+			msgs = append(msgs, "Client netcheck: "+warn.String())
+		}
+	}
+
+	if b.Deployment.HealthReport == nil {
+		msgs = append(msgs, "Deployment health check missing from bundle!")
+	} else {
+		if b.Deployment.HealthReport.AccessURL.Error != nil {
```

Comment:
Maybe we can improve the design a bit -
Right now the Bundle must be aware of HealthReport internals, so whenever somebody modifies the report, they need to adjust the bundle logic too. Maybe we should add a new method to `HealthReport`, `Summarize(b *Bundle)` and let the health report decide what should be shared or not?

Reply:
I might move this to a separate PR; It feels like the precursor to a `coder healthcheck` CLI command.

Reply:
https://github.com/coder/coder/pull/12963